src/lib/crypto/kimchi_pasta_snarky_backend/field.ml

```diff
@@ -203,10 +203,16 @@ module Make (F : Input_intf) :
 
       let of_yojson j =
         match j with
-        | `String h ->
-            Ok (of_bigint (Bigint.of_hex_string h))
+        | `String s ->
+            let parsed_bigint =
+              if String.is_prefix ~prefix:"0x" s then Bigint.of_hex_string s
+              else
+                (* NOTE: we're dealing with a older precomputed block *)
+                Bigint.of_decimal_string s
+            in
+            Ok (of_bigint parsed_bigint)
         | _ ->
-            Error "expected hex string"
+            Error "expected a hex string or a decimal string(for old PCBs)"
```

Comment:
Small nitpick (feel free to change in separate PR if this would delay things too much) - these field elements can appear in more than just precomputed blocks, so the error might not want to mention PCBs in particular